### PR TITLE
chore(KFLUXVNGD-38): remove maintainers delete permissions

### PR DESCRIPTION
**Pull request by yftacherzog**
Remove Project* delete permissions from maintainers, as maintainers should not have permissions to delete applications or components.

### CODE REVIEW

File: components/sandbox/tiers/production/appstudio-env/tiertemplate-appstudio-env-admin-37575401-37575401.yaml [hunk 1]
```diff
@@ -12,7 +12,8 @@ spec:
   template:
     apiVersion: template.openshift.io/v1
     kind: Template
-    metadata: {}
+    metadata:
+      creationTimestamp: null
```
Comment (hugares) [inline components/sandbox/tiers/production/appstudio-env/tiertemplate-appstudio-env-admin-37575401-37575401.yaml:16]: all those creationTimestamp chages should not be there. This is a bug in ksctl so you need to build a custom version of ksctl including this change: https://github.com/kubesaw/ksctl/pull/50

I can re-generate the tiers for you and update the PR if you want.

Comment (yftacherzog) [reply components/sandbox/tiers/production/appstudio-env/tiertemplate-appstudio-env-admin-37575401-37575401.yaml]: @hugares  I redid the change after installing the tool from that branch. Please take another look and let me know.

**Review by hugares — APPROVED** [review]
/lgtm

File: components/sandbox/tiers/production/appstudio-env/tiertemplate-appstudio-env-admin-37575401-37575401.yaml (hunk 1 above)
Comment (hugares) [reply components/sandbox/tiers/production/appstudio-env/tiertemplate-appstudio-env-admin-37575401-37575401.yaml]: looks good, Thanks!

**Pull request merged**